charts/hivemq-platform/templates/hivemq-custom-resource.yml

```
@@ -2,23 +2,24 @@
 {{- $hasLicense := ( include "hivemq-platform.has-license" . ) -}}
 {{- $hasAdditionalVolumes := ( include "hivemq-platform.has-additional-volumes" . ) -}}
 {{- $hasAdditionalVolumeMounts := ( include "hivemq-platform.has-additional-volume-mounts" . ) -}}
-{{- $defaultMetricsPort := ( include "hivemq-platform.metrics-port" . ) -}}
+{{- $metricsPort := ( include "hivemq-platform.metrics-port" . ) -}}
+{{- $hasMetricService := ( include "hivemq-platform.has-default-metrics-service" . ) }}
```

I never get these subtle differences, but before the last include line ended with `-}}`, not `}}`. Should this be aligned?
```suggestion
{{- $hasMetricService := ( include "hivemq-platform.has-default-metrics-service" . ) -}}
```

---

Good catch! I'm not 100% sure either whether this is needed in this case, but added that so it's aligned with the rest